stronk/schemas/workout/query.py

```diff
@@ -0,0 +1,34 @@
+
+import graphene
+
+from stronk.models.program_workouts import ProgramWorkouts as ProgramWorkoutsModel
+from stronk.models.workout import Workout as WorkoutModel
+
+from stronk.schemas.workout.type import Workout as WorkoutType
+from stronk.schemas.workout.type import ProgramWorkouts as ProgramWorkoutsType
+
+from werkzeug.exceptions import BadRequest, NotFound
+
+
+class Query(graphene.ObjectType):
+    workouts = graphene.List(WorkoutType,
+                             program_id=graphene.NonNull(graphene.Int))
+    workout = graphene.Field(WorkoutType,
+                             id=graphene.NonNull(graphene.Int))
+
+    def resolve_workouts(root, info, program_id: int):
+        """Retrieve all workouts for a program
+        """
+        query = WorkoutType.get_query(info)
+        workouts = query.join(ProgramWorkoutsModel).filter(
+            ProgramWorkoutsModel.program_id == program_id)
+        return workouts
+
+    def resolve_workout(root, info, id: int = None):
+        """Search for a specific workout
+        """
+        # sqlachemy query bindings
+        query = WorkoutType.get_query(info)
+
+        # TODO check for permissions for each workout
```

Comment:
We should probably make a ticket for this if we're not gonna tackle this is this PR

Reply:
Will add ticket :)

Reply:
Imma just remove the todo in the code and refer to the jira ticket since this is something that needs to be done for alot of resolvers later.